src/Map/src/Bridge/Google/src/GoogleOptions.php

```diff
@@ -46,6 +46,11 @@ public function mapId(?string $mapId): self
         return $this;
     }
 
+    public function hasMapId(): bool
+    {
+        return null !== $this->mapId;
+    }
```

Comment:
Maybe we can make properties public..?

Reply:
As @nicolas-grekas explained me (more than once.. i'm a slow learner), it's harder to maintain BC with property or named arguments than good old-fashioned methods :)

Reply:
(yeah I felt the same when I saw the migration from `KernelTestCase::$container` to `KernelTestCase::getContainer()` :p)